### PR TITLE
Feature/agenda: Allow to connect to office 365 account

### PR DESCRIPTION
**Pull request by aelbenney**

### CODE REVIEW

File: agenda-webapps/src/main/webapp/vue-app/agenda-connectors/agendaOfficeConnector.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+      storeAuthStateInCookie: false, // Set this to 'true' if you are having issues on IE11 or Edge
+    }
+  },
+  graphConfig: {
+    graphMeEndpoint: 'https://graph.microsoft.com/v1.0/me',
+    graphCalendarEventsEndpoint: 'https://graph.microsoft.com/v1.0/me/calendar/calendarView?'
+  },
+  loginRequest: {
+    scopes: ['Calendars.Read'],
+    redirectUri: 'http://localhost:8080/',
```
Comment (plamarque) [inline agenda-webapps/src/main/webapp/vue-app/agenda-connectors/agendaOfficeConnector.js]: hardcoded?

File: agenda-webapps/src/main/webapp/vue-app/agenda-connectors/agendaOfficeConnector.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+        events.forEach(event => {
+          event.allDay = !!event.isAllDay;
+          const StartDate = event.start.dateTime && new Date(`${event.start.dateTime}Z`);
+          const EndDate = event.end.dateTime && new Date(`${event.end.dateTime}Z`);
+          event.start = StartDate || event.start.date;
+          event.end = EndDate  || event.start.date;
+          event.summary = event.subject;
+          event.location = event.location.displayName;
+          event.type = 'remoteEvent';
+          event.color = '#FFFFFF';
```
Comment (plamarque) [inline agenda-webapps/src/main/webapp/vue-app/agenda-connectors/agendaOfficeConnector.js]: hardcoded?

File: agenda-webapps/src/main/webapp/vue-app/agenda-connectors/agendaOfficeConnector.js (hunk 1 above)
Comment (aelbenney) [reply agenda-webapps/src/main/webapp/vue-app/agenda-connectors/agendaOfficeConnector.js]: useless variable! to be cleaned.

File: agenda-webapps/src/main/webapp/vue-app/agenda-connectors/agendaOfficeConnector.js (hunk 2 above)
Comment (aelbenney) [reply agenda-webapps/src/main/webapp/vue-app/agenda-connectors/agendaOfficeConnector.js]: Those are the values we want (type and color) for remote events.

File: agenda-webapps/src/main/webapp/vue-app/agenda-connectors/agendaOfficeConnector.js (hunk 1 above)
Comment (boubaker) [reply agenda-webapps/src/main/webapp/vue-app/agenda-connectors/agendaOfficeConnector.js]: done

**Pull request merged**